src/include/stir/recon_buildblock/BackProjectorByBin.h

```diff
@@ -127,11 +156,11 @@ class BackProjectorByBin :
   virtual void check(const ProjDataInfo& proj_data_info, const DiscretisedDensity<3,float>& density_info) const;
   bool _already_set_up;
 
+  //! The density ptr set with set_up()
```

Comment:
Make clear if this is a new object (created with a clone) or not (I hope so, as otherwise the current (non-OpenMP) code would override its content).